scss/inc/_select2.scss

```diff
@@ -147,6 +160,19 @@
     border-top: 0;
 
     border-radius: 0 0 2px 2px;
+    @include rtl {
+        direction: rtl;
+    }
+    @include ltr {
+        direction: ltr;
+    }
+
+    &.rtl {
+        direction: rtl;
+    }
+    &.ltr {
+        direction: ltr;
+    }
```

Comment:
Do we really need it?
Seems to be you did a good job here: https://github.com/oat-sa/extension-tao-itemqti/pull/1958/files

Reply:
Yes, this is for the dropbox at the bottom of the document. It is outside the component, and the other CSS can't make any difference here.
![image](https://user-images.githubusercontent.com/11692751/143543508-9709f40e-563b-4972-8bd3-4efdfe004f53.png)